z_infra_provisioning/cloud_infra_center/ocp_upi/README.md

```diff
@@ -50,7 +50,7 @@ of this method of installation.
 The file `inventory.yaml` contains the variables most likely to need customisation.
 
 For a successful IBM® Cloud Infrastructure Center UPI installation (3 worker nodes) it is required:
```

Comment:
I found a "," is missing before "it is required", can you add it? Thanks~

Reply:
Okay, I have added it.

Reply:
"....(3 worker nodes) it is required:"  should read "....(3 worker nodes) are required:"

Reply:
Thanks, I've updated it.